test/test_auth.py

```diff
@@ -204,6 +215,8 @@ def test_gssapi_simple(self):
 
     @ignore_deprecations
     def test_gssapi_threaded(self):
+        if not _IS_SYNC:
```

Comment:
Use `@async_client_context.require_sync` as a decorator for sync-only tests instead.